### PR TITLE
[Docs] update Japanese translation of feature_pointing_device.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

update Japanese translation of docs/feature_pointing_device.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

**Review by s-show — APPROVED** [review]

**Review by shelaf — APPROVED** [review]

File: docs/ja/feature_pointing_device.md [hunk 1]
```diff
@@ -24,14 +24,18 @@ report_mouse_t (ここでは "mouseReport") が以下のプロパティを持つ
...
-* `mouseReport.buttons` - これは uint8_t で、上位の5ビットを使っています。これらのビットはマウスボタンの状態を表します - ビット 3 はマウスボタン 5、ビット 7 はマウスボタン 1 です。
+* `mouseReport.buttons` - これは uint8_t で、8ビット全てを使っています。これらのビットはマウスボタンの状態を表します - ビット 0 はマウスボタン 1、ビット 7 はマウスボタン 8 です。
 
 マウスレポートに必要な変更を行ったら、それを送信する必要があります:
 
 * `pointing_device_send()` - マウスレポートをホストに送信し、レポートをゼロにします。
 
 マウスレポートが送信されると、x、y、v、h のいずれの値も 0 に設定されます (これは `pointing_device_send()` で行われます。この挙動を回避するためにオーバーライドすることができます)。このように、ボタンの状態は持続しますが、動きは1度だけ起こります。さらにカスタマイズするために、`pointing_device_init` と `pointing_device_task` のどちらもオーバーライドすることができます。
 
+さらに、デフォルトでは、`pointing_device_send()` はレポートが実際に変更された場合のみレポートを送信します。これにより、マウスレポートが継続的に送信されなくなり、ホストシステムが起動されたままになります。この動作は、独自の `pointing_device_send()` 関数を作成することで変更できます。
```
Comment (mtei) [inline docs/ja/feature_pointing_device.md:35]: 無駄なマウスレポートを抑制するのは、ホストがスリープに入れなくなる事態を防ぐためだと思うので、文意として以下ではないでしょうか？
```suggestion
さらに、デフォルトでは、`pointing_device_send()` はレポートが実際に変更された場合のみレポートを送信します。これにより、マウスレポートが継続的に送信されてホストシステムが起動されたままになることを防ぎます。この動作は、独自の `pointing_device_send()` 関数を作成することで変更できます。
```

File: docs/ja/feature_pointing_device.md [hunk 2]
```diff
@@ -24,14 +24,18 @@ report_mouse_t (ここでは "mouseReport") が以下のプロパティを持つ
...
+
+また、`has_mouse_report_changed(new, old)` 関数を使って、レポートが変更されたかどうかを確認します。
```
Comment (mtei) [inline docs/ja/feature_pointing_device.md:37]: @drashna 

There is the following text in feature_pointing_device.md.

> Also, you use the `has_mouse_report_changed(new, old)` function to check to see if the report has changed.

Does the above sentence have the same meaning as the following sentence?

> Also, you can use the `has_mouse_report_changed(new, old)` function to check to see if the report has changed.

Comment (drashna) [reply docs/ja/feature_pointing_device.md:37]: Yup. 

And you want to check it.  If the report hasn't changed and you send it, the host considers that mouse activity, and will keep the screen and system active.  So if you replace the `pointing_device_send()` function with a custom implementation, you want to include this check, manually.

Comment (mtei) [reply docs/ja/feature_pointing_device.md:37]: Thank you!

Comment (mtei) [reply docs/ja/feature_pointing_device.md:37]: 訳注を追加しました。
```suggestion
また、`has_mouse_report_changed(new, old)` 関数を使って、レポートが変更されたかどうかを確認できます。(訳注：独自の `pointing_device_send()` 関数を作成する場合でも、その中で `has_mouse_report_changed(new, old)` 関数でチェックして、デフォルトの `pointing_device_send()` と同様に、無駄なレポートをしないようにして、ホストシステムがスレープ状態に入れるようにしておくのが良いでしょう。)
```

File: docs/ja/feature_pointing_device.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/feature_pointing_device.md]: ご指摘ありがとうございます！

File: docs/ja/feature_pointing_device.md (hunk 2 above)
Comment (umi-umi) [reply docs/ja/feature_pointing_device.md]: 訳例ありがとうございます。唐突に has_mouse_report_changed の話が出てきて戸惑ったのですが、文意を考えればそういう事を伝えたいということだったんですね。

**Review by mtei — APPROVED** [review]

**Comment by mtei** [comment]
I think this is ready to merge.

**Comment by mtei** [comment]
Thanks!

**Pull request merged**